docs/change-log/2025-10-15-modal-file-upload.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: "Adding More Modal Components!"
+title: "Modal File Upload"
```

Comment:
```suggestion
title: "Introducing the File Upload component in Modals"
```
something a bit more verbose like this would fit better with the other changelog names imo